community/GOVERNANCE.md

```diff
@@ -1,18 +1,35 @@
 # Argo Governance
+This document outlines the governance for the overall Argo Project and the four sub-projects contained within.  
 
-Describes the roles and responsibilities and governing structure of the Argo Community.
+## Roles and Membership
+Roles are described in the [Community Membership](https://github.com/argoproj/argoproj/blob/master/community/membership.md) document
+For the purpose of governance, reviewers, approvers and leads are considered maintainers.
```

Comment:
May I request myself to be added as a reviewer to https://github.com/argoproj/argoproj/blob/master/OWNERS ?

Reply:
There's separate process for that. See https://github.com/argoproj/argoproj/blob/master/community/membership.md#reviewer

Reply:
Thanks @terrytangyuan , the process in there adheres well to code contributors / reviewers but not necessarily to folks who may be playing other roles in the project related to project design/architecture. I wonder if we should make a separate proposal on that?

Reply:
Perhaps @hblixt @edlee2121 and others might offer some advice here. Perhaps outlining your contributions towards design/architecture in your request and discuss with your sponsor before submitting your request would help. Non-code contribution is definitely valuable. In the meantime, I'd say at least the knowledge/familiarity of the code base is very important and proves at a certain degree whether one can provide helpful review feedback.

Reply:
💯 !
I would definitely follow the process for being added as a reviewer. My question was more of a general one 🚢 

( Personally I do have a fair amount of https://github.com/argoproj/argo-cd/pulls and demonstrated familiarity of the code, so I'm good )

Reply:
There has been a discussion in the bootstrap committee about expanding maintainers with non-code writing contributors, to foster more inclusion and diverse contributions.  
As @terrytangyuan points out, there's not really a notion of that in the membership file right now though, so that would need to be addressed.